src/daft-connect/src/spark_analyzer.rs

```diff
@@ -477,7 +476,8 @@ impl SparkAnalyzer<'_> {
             let chunk = match ss {
                 StreamState::Some(chunk) => chunk,
                 StreamState::Waiting => {
-                    bail!("StreamReader is waiting for data, but a chunk was expected. This likely indicates that the spark provided data is incomplete.")
+
```

Comment:
Did we run the linter on this commit? Usually it catches and removes empty lines...

Reply:
the formatter/linter is kinda bad at picking up newlines and such when using macros